influxql/iterator.gen.go.tmpl

```diff
@@ -370,6 +371,84 @@ func (itr *{{.name}}LimitIterator) Next() *{{.Name}}Point {
 	}
 }
 
+// {{.name}}FillIterator represents a fill iterator that processes {{.name}} values.
+type {{.name}}FillIterator struct {
+	name  string
+	input {{.Name}}Iterator
+	buf   *{{.Name}}Point
+	prev  *{{.Name}}Point
+	next  int64
+	opt   IteratorOptions
+	fill  func (*{{.name}}FillIterator) {{.Type}}
+}
+
+func new{{.Name}}FillIterator(input {{.Name}}Iterator, opt IteratorOptions) *{{.name}}FillIterator {
+	measurement := opt.Sources[0].(*Measurement)
+	var fill func (*{{.name}}FillIterator) {{.Type}}
+	switch opt.Fill {
+	case NullFill:
+		fill = func (*{{.name}}FillIterator) {{.Type}} {
+			return {{.Nil}}
+		}
+	case NumberFill:
+		v, ok := opt.FillValue.({{.Type}})
+		if !ok {
+			panic("invalid fill value for type {{.Type}}")
```

Comment:
Can you use `fmt.Sprintf` with `%T` so you can print the type received? It'll make debugging easier if we hit the panic.